python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -665,11 +665,21 @@ def get_runs_feed_entries(
     )
 
 
-def get_runs_feed_count(graphene_info: "ResolveInfo", filters: Optional[RunsFilter]) -> int:
-    should_fetch_backfills = _filters_apply_to_backfills(filters) if filters else True
+def get_runs_feed_count(
+    graphene_info: "ResolveInfo", filters: Optional[RunsFilter], include_runs_from_backfills: bool
+) -> int:
+    # the UI is oriented toward showing runs that are part of a backfill, but the backend
+    # is oriented toward excluding runs that are part of a backfill, so negate include_runs_in_backfills
+    # to get the value to pass to the backend
+    exclude_subruns = not include_runs_from_backfills
```

Comment:
should we just do `include_subruns` as the negation of `exclude_subruns`, just for consistency?